GDSCommon-Demo/GDSCommon-Demo/Mocks/Screens.swift

```diff
@@ -85,7 +85,9 @@ enum Screens: String, CaseIterable {
             view.isModalInPresentation = true
             return view
         case .gdsListOptions:
-            return ListOptionsViewController(viewModel: MockListViewModel())
+            return ListOptionsViewController(viewModel: MockListViewModel(dismissAction: {
+                navigationController.popToRootViewController(animated: true)
```

Comment:
Can we have this as a trailing closure and save a set of brackets? Maybe not possible if its within the vc brackets

Reply:
I've resolved this (partially anyway) by moving the complexity into the view controller initialisation extension file.